app/helpers/application_helper.rb

```diff
@@ -6,8 +6,4 @@ def url_checker(post)
   def time_and_date(post)
     post.created_at.strftime("%d/%m/%Y at %H:%M")
   end
-
-  def post_presenter(post)
-    simple_format("#{url_checker(post)} <br>Posted on #{time_and_date(post)}")
-  end
```

Comment:
Method added back in and other suggestions updated. Also merged with new master although comments are not styled overly nicely. Adam is planning to jump on this once this is pushed to master.